Vungle/VungleAdapter/ALVungleMediationAdapter.m

```diff
@@ -115,9 +116,29 @@ - (void)initializeWithParameters:(id<MAAdapterInitializationParameters>)paramete
     
     if ( [ALVungleInitialized compareAndSet: NO update: YES] )
     {
+        [self initVungleSDK:parameters completionHandler:completionHandler];
+    }
+    else
+    {
+        completionHandler(ALVungleIntializationStatus, nil);
+    }
+}
+
+- (void)initVungleSDK:(id<MAAdapterParameters>)parameters completionHandler:(void (^)(MAAdapterInitializationStatus, NSString *_Nullable))completionHandler {
```

Comment:
What's the point of having the `, NSString *_Nullable` in the completionHandler if it's not used in any of the blocks during the ad load calls?